src/integrationTest/resources/mappings/case-incomplete-creation-request.json

```diff
@@ -0,0 +1,59 @@
+{
+  "event": {
+    "id": "incompleteApplicationReceived",
+    "summary": null,
+    "description": "Case created from exception record"
+  },
+  "data": {
+    "appeal": {
+      "mrnDetails": {
+        "dwpIssuingOffice": null,
+        "mrnDate": null,
+        "mrnLateReason": null,
+        "mrnMissingReason": null
+      },
+      "appellant": {
+        "name": {
+          "title": "Mr",
+          "firstName": "John",
+          "lastName": null
+        },
+        "address": { },
+        "contact": {
+          "email": null,
+          "phone": null,
+          "mobile": null
+        },
+        "identity": {
+          "dob": null,
+          "nino": null
+        },
+        "appointee": null
+      },
+      "benefitType": {
+        "code": "Employment Support Allowance",
+        "description": null
+      },
+      "hearingOptions": {
+        "wantsToAttend" : "Yes",
+        "wantsSupport" : "No",
+        "languageInterpreter": "No",
+        "arrangements" : []
+      },
+      "appealReasons": null,
+      "rep": {
+        "hasRepresentative": "No",
+        "organisation": null,
+        "name": null,
+        "address": null,
+        "contact": null
+      },
+      "signer": null,
+      "hearingType": "oral"
+    },
+    "sscsDocument" : [ ]
+  },
+  "security_classification" : "PUBLIC",
+  "event_token" : "test_event_token",
+  "ignore_warning" : false
```

Comment:
should this not be true when caseworker has selected to ignore warning ?

Reply:
sorry for confusion this is correct. I thought this is the ignore warning field value sent by ccd as part of callback